eth/accountmanager.go

```diff
@@ -25,19 +25,19 @@ type AccountManager interface {
 	Unlock(passphrase string) error
 	Lock() error
 	CreateTransactOpts(gasLimit uint64, gasPrice *big.Int) (*bind.TransactOpts, error)
-	SignTx(signer types.Signer, tx *types.Transaction) (*types.Transaction, error)
+	SignTx(tx *types.Transaction) (*types.Transaction, error) // SignTx is configured with EIP155Signer by default
```

Comment:
Don't think we need this comment as the caller can configure `AccountManager` with any `Signer` - we just happen to always configure it with `EIP155Signer`.

Reply:
fixed in aee8f653